src/docs/README.md

```diff
@@ -2,6 +2,7 @@
 
 * Introduction
   * [Overview](introduction/overview.md)
+  * [Upgrading to Stencil v3](introduction/upgrading-to-stencil-three.md)
```

Comment:
Up for discussion. I added it here to:
1. align with what the Ionic Framework here did
2. placing this in the "references" section seemed to hurt the find-ability of this guide